src/addresses.ts

```diff
@@ -0,0 +1,46 @@
+import { HexString } from "./types";
+import { generateHexString } from "./index";
+
+const preFabAddr = "1Ff482D42D8727258A1686102Fa4ba925C";
+
+/**
+ * Generate a wallet address
+ */
+export const generateEthereumAddress = (): HexString => {
+  return generateHexString(20);
+};
+
+/**
+ * Get a prefabricated social identity address compatible with other prefab data
+ * Prefab social identity address is `0xCODE0000`
+ */
+export const getPrefabAddress = (index: number): HexString => {
+  return ["0x", preFabAddr, zeroFill(index, 5)].join("");
```

Comment:
Are we still prefixing '0x' in front of hex strings? I don't think this is necessary.  I left it because this is what we've been doing.

Reply:
I'm a fan of the prefix. A lot of eth libraries require it to tell the difference between string and hex (as hex might be processed differently).

I also wonder:
- Should make sure these zeroFill to 40 char to be a full length address?
- Should we make sure these are "legal" addresses (or try to make sure they are bad?) https://www.rfctools.com/ethereum-address-test-tool/